TI_Practice1/Circle.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Practice1
+{
+    class Circle
+    {
+        public Circle(double val)
+        {
+            radius = val;
+        }
+
+        readonly double radius;
+        double resp;
+
+        public double GetArea()
+        {
+            resp = Math.PI * Math.Pow(radius, 2);
+            return resp;
```

Comment:
The "resp" variable is not needed here, you can simply use
return Math.PI * Math.Pow(radius, 2);

It is better to get rid of all the unnecessary variables.

Furthermore, in this particular case, you're using a class field. This is not correct because a class field represents some semantic property. E.g. class Car will have engine, wheels, etc. fields.
All the other subsidiary values should be in the local variables.

Reply:
DONE. Can we discuss class fields and properties usage during the next practice/lecture